che-mount/entrypoint.sh

```diff
@@ -93,45 +113,105 @@ trap 'stop_sync' SIGHUP SIGTERM SIGINT
 init_logging
 init_global_variables
 parse_command_line "$@"
-status=$?
-if [ $status -ne 0 ]; then
-    exit 1
+[ "$(ls -A /mnthost )" ] && EMPTY=false || EMPTY=true
+if [ $EMPTY = false ]; then
+    warn "(che mount): Local folder is not empty. This could overwrite folders/files on local desktop machine or remote workspace machine. Are you sure you want to continue?"
+    read yn
+    case $yn in
+        [Yy]* ) break;;
+        [Nn]* ) exit;;
+        * ) info "(che mount): Please answer yes or no.";;
+    esac
 fi
-
-docker run --rm  -v /var/run/docker.sock:/var/run/docker.sock eclipse/che-action:${CHE_VERSION} get-ssh-data ${WORKSPACE_NAME} ${COMMAND_EXTRA_ARGS} > $HOME/env
+docker run --rm  -v /var/run/docker.sock:/var/run/docker.sock $REPO/che-action:${CHE_VERSION} get-ssh-data ${WORKSPACE_NAME} ${COMMAND_EXTRA_ARGS} > $HOME/env
 if [ $? -ne 0 ]; then
     error "ERROR: Error when trying to get workspace data for workspace named ${WORKSPACE_NAME}"
     echo "List of workspaces are:"
-    docker run --rm  -v /var/run/docker.sock:/var/run/docker.sock eclipse/che-action:${CHE_VERSION} list-workspaces
+    docker run --rm  -v /var/run/docker.sock:/var/run/docker.sock $REPO/che-action:${CHE_VERSION} list-workspaces ${COMMAND_EXTRA_ARGS}
```

Comment:
I am -1 on having the $REPO option for users to set.  We are already having problems with too many image names in variables.  This is going to get removed as part of the ocnversion into new Che CLI